katz/src/main/kotlin/katz/predef.kt

```diff
@@ -0,0 +1,28 @@
+package katz
+
+import katz.typeclasses.*
+import katz.typeclasses.HK
+import katz.typeclasses.Monad
+import katz.typeclasses.MonadError
+import kotlin.coroutines.experimental.startCoroutine
+
+typealias HK<F, A> = katz.typeclasses.HK<F, A>
+typealias Functor<F> = katz.typeclasses.Functor<F>
+typealias Applicative<F> = katz.typeclasses.Applicative<F>
+typealias ApplicativeError<F, E> = katz.typeclasses.ApplicativeError<F, E>
+typealias Monad<F> = katz.typeclasses.Monad<F>
+typealias MonadError<F, E> = katz.typeclasses.MonadError<F, E>
```

Comment:
by having these aliases here we can have the code organized in whichever package structure we want and provide users with an uber import `import katz.*` and also give them the option to import them a la carte.